src/transformers/modeling_utils.py

```diff
@@ -53,7 +53,20 @@ def forward(self, input):
             return input
 
 
-class PreTrainedModel(nn.Module):
+class ModuleUtilsMixin:
```

Comment:
Why do we need to use a `MixIn` here? I know we've used that in `test` but I still think it's a practice that should be generalized only after careful discussion. It's often debated in the python world.